test/api-client/api-client.captions.spec.ts

```diff
@@ -0,0 +1,54 @@
+import { RevAiApiClient } from '../../src/api-client';
```

Comment:
I also took this opportunity to clean up and split up our client tests :)

Reply:
I can't see the diffs as well when every line is moved to a new file, but I enjoy better separation of the tests